base/system/winlogon/sas.c

```diff
@@ -46,6 +46,12 @@ static BOOL ExitReactOSInProgress = FALSE;
 
 LUID LuidNone = {0, 0};
 
+typedef struct tagLOGON_SOUND_DATA
```

Comment:
```suggestion
typedef struct _LOGON_SOUND_DATA
```

Reply:
I would like to keep it consistent with `LOGOFF_SHUTDOWN_DATA` for now.